image_cloudevents/README.md

```diff
@@ -1 +1,96 @@
+       - **TOKEN** = {your dkube authentication token}
+       - **DKUBE_URL** = {your dkube url}
+5. Run all the cells. This will create all the dkube resources required for this example automatically.
+6. MINIO_KEY and MINIO_SECRET values will be filled automatically by the example with SDK call, these values can also be obtained by running the following commands on the DKube setup where the prediction deployment is running. Provide the creds manually if the user is neither PE nor Operator on the remote cluster.
+    - `kubectl get secret -n dkube-infra cloudevents-minio-secret -o jsonpath="{.data.AWS_ACCESS_KEY_ID}" | base64 -d`
+    - `kubectl get secret -n dkube-infra cloudevents-minio-secret -o jsonpath="{.data.AWS_SECRET_ACCESS_KEY}" | base64 -d`
+
+### Launch IDE (Outside Dkube)
+
+#### Note: Follow the instructions if you are running Notebook IDE outside DKube, for example in VSCode. This is the case with minimal DKube. 
+
+1. Download [resources.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/image_cloudevents/resources.ipynb)
+2. Open the notebook and fill the details in the first cell.
+    - **DKUBE_IP** = {IP address of the DKube where the prediction deployment is running}
+    - **DKUBEUSERNAME** = {your dkube username}
+    - **MODELMONITOR_NAME** = {your model monitor name}
+    - **TOKEN** = {your dkube authentication token}
+    - **DKUBE_URL** = {your dkube url}
+    - **MINIO_KEY** = {MINIO access key of Dkube setup where the prediction deployment is running}
+    - **MINIO_SECRET_KEY** = {MINIO access secret key of Dkube setup where the prediction deployment is running}
+3. Run all the cells.
+
+**Note:** In case the monitor is being created on minimal DKube fill the **DKUBEUSERNAME**, **TOKEN**, and **DKUBE_URL** for the minimal DKube. 
+
+## Section 2: Insurance Model Training (Required to deploy model)
+
+#### Note: This uses DKube Runs, Kubeflow Pipelines and KfServing. It requires full Dkube installed. 
+
+1. From **workspace/chest-xray/image_cloudevents** open **train.ipynb** to build the pipeline.
+2. The pipeline includes preprocessing, training and serving stages. Run all cells
+     - **preprocessing**: the preprocessing stage generates the dataset (either training-data or retraining-data) depending on user choice.
+     - **training**: the training stage takes the generated dataset as input, train a sgd model and outputs the model.
+     - **serving**: The serving stage takes the generated model and serve it with a predict endpoint for inference.
+3. Verify that the pipeline has created the following resources
+     - Datasets: 'insurance-training-data' with version v2.
+     - Model: 'insurance-model' with version v2
+
+## Section 3: Modelmonitoring
+DKube provides Python SDK for creating a modelmonitor programmatically. You could also choose to create a modelmonitor from the DKube UI. This example cuurently support creating monitor using SDK. 
```

Comment:
Is there an image example with UI steps for creating modelmonitor? We need to have at least 1.

I see there is README.ui.md. What is it for?

Reply:
The readme.ui.md is having steps to make model monitor from UI, as we are doing for other examples.